firmware/controllers/engine_cycle/main_trigger_callback.cpp

```diff
@@ -352,7 +352,7 @@ void mainTriggerCallback(uint32_t trgEventIndex, efitick_t edgeTimestamp) {
 	ScopePerf perf(PE::MainTriggerCallback);
 
 	if (noFiringUntilVvtSync(engineConfiguration->vvtMode[0]) 
-		&& !engine->triggerCentral.vvtState[0][0].getShaftSynchronized()) {
```

Comment:
now that we are fixing recent changes It sounds like we need more test coverage

Reply:
Done, looks like the map cam test was sort of broken anyway :)